lib/trino-array/src/test/java/io/trino/array/TestSliceBigArray.java

```diff
@@ -14,88 +14,87 @@
 package io.trino.array;
 
 import io.airlift.slice.Slice;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.Test;
+import org.junit.jupiter.api.Test;
 
 import static io.airlift.slice.SizeOf.instanceSize;
 import static io.airlift.slice.SizeOf.sizeOf;
 import static io.airlift.slice.Slices.wrappedBuffer;
-import static org.testng.Assert.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
 
-@Test(singleThreaded = true)
 public class TestSliceBigArray
 {
     private static final long BIG_ARRAY_INSTANCE_SIZE = instanceSize(SliceBigArray.class) + new ReferenceCountMap().sizeOf() + new ObjectBigArray<Slice>().sizeOf();
     private static final long SLICE_INSTANCE_SIZE = instanceSize(Slice.class);
     private static final int CAPACITY = 32;
     private final byte[] firstBytes = new byte[1234];
     private final byte[] secondBytes = new byte[4567];
-    private SliceBigArray sliceBigArray;
-
-    @BeforeMethod
-    public void setup()
-    {
-        sliceBigArray = new SliceBigArray();
-        sliceBigArray.ensureCapacity(CAPACITY);
-    }
 
     @Test
     public void testSameSliceRetainedSize()
     {
+        SliceBigArray sliceBigArray = new SliceBigArray();
+        sliceBigArray.ensureCapacity(CAPACITY);
+
         // same slice should be counted only once
         Slice slice = wrappedBuffer(secondBytes, 201, 1501);
         for (int i = 0; i < CAPACITY; i++) {
             sliceBigArray.set(i, slice);
-            assertEquals(sliceBigArray.sizeOf(), BIG_ARRAY_INSTANCE_SIZE + sizeOf(secondBytes) + SLICE_INSTANCE_SIZE);
+            assertThat(sliceBigArray.sizeOf()).isEqualTo(BIG_ARRAY_INSTANCE_SIZE + sizeOf(secondBytes) + SLICE_INSTANCE_SIZE);
         }
 
         // adding a new slice will increase the size
         slice = wrappedBuffer(secondBytes, 201, 1501);
         sliceBigArray.set(3, slice);
-        assertEquals(sliceBigArray.sizeOf(), BIG_ARRAY_INSTANCE_SIZE + sizeOf(secondBytes) + SLICE_INSTANCE_SIZE * 2);
+        assertThat(sliceBigArray.sizeOf()).isEqualTo(BIG_ARRAY_INSTANCE_SIZE + sizeOf(secondBytes) + SLICE_INSTANCE_SIZE * 2);
     }
 
     @Test
     public void testNullSlicesRetainedSize()
     {
+        SliceBigArray sliceBigArray = new SliceBigArray();
```

Comment:
moving SliceBigArray from setup to a method should be extracted to a preparatory commit